src/common/dpi_aware.cpp

```diff
@@ -37,4 +37,9 @@ void DPIAware::Convert(HMONITOR monitor_handle, int &width, int &height) {
     width = width * dpi_x / DEFAULT_DPI;
     height = height * dpi_y / DEFAULT_DPI;
   }
-}
\ No newline at end of file
+}
+
+void DPIAware::EnableDPIAwarenessForThisProcess()
+{
```

Comment:
This file uses the "curly brace on the same line" style.

Reply:
ah, thanks! it would be awesome to integrate .clang-format for those kinds of issues :)